sources/AppBundle/Controller/MemberShipController.php

```diff
@@ -265,6 +265,16 @@ public function payboxCallbackAction(Request $request)
         return new Response();
     }
 
+    public function payboxRedirectAction(Request $request)
+    {
+        $payboxResponse = PayboxResponseFactory::createFromRequest($request);
+
+        return $this->render(':site/company_membership:paybox_redirect.html.twig', [
```

Comment:
Idéalement, en faire une classe `PayboxRedirectAction` et faire un `return new Response($this->twig->render(...` pour éviter l'héritage du Controller Symfony

Reply:
Modifier